assets/tests/testHelpers/mockHelpers.ts

```diff
@@ -79,3 +80,14 @@ export const mockFullStoryEvent = (): void => {
     value: { event: jest.fn(), identify: jest.fn() },
   })
 }
+
+export const mockTileUrls = (): void => {
+  ;(tilesetUrlForType as jest.Mock).mockImplementation((type: TileType) => {
+    switch (type) {
+      case "base":
+        return "test_base_url.com/{z}/{x}/{y}"
+      case "satellite":
+        return "test_satellite_url.com/{z}/{x}/{y}"
```

Comment:
suggestion: Can we make these url's impossible to resolve?
maybe removing only the tld is enough and should still function?

```suggestion
      case "base":
        return "test_base_url/{z}/{x}/{y}"
      case "satellite":
        return "test_satellite_url/{z}/{x}/{y}"
```

I could see localhost being an option if that doesn't work, but I'd rather that there's a URL that couldn't resolve.

Reply:
Updated in https://github.com/mbta/skate/pull/2118/commits/b33e2d2798df639c1b7907229277d8d65b72544e!